config.gui.prod.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
 	phaseControl: false,
 	apertureControl: true
```

Comment:
There is a comma missing at the end of this line, you get a syntax error when you build the project ...